tests/parsely/test-mu-parsely-integration.php

```diff
@@ -337,13 +337,13 @@ public function test_parsely_configs_for_managed_mode() {
 		$configs = Parsely_Loader_Info::get_configs();
 
 		// Assert.
-		$this->assertEquals( $configs, array(
+		$this->assertEquals( array(
 			'is_pinned_version'            => has_filter( 'wpvip_parsely_version' ),
-			'site_id'                      => is_latest_version() ? 'site_id_value' : '',
-			'have_api_secret'              => is_latest_version() ? true : false,
+			'site_id'                      => 'site_id_value',
+			'have_api_secret'              => true,
 			'is_javascript_disabled'       => false,
 			'is_autotracking_disabled'     => false,
-			'should_track_logged_in_users' => is_latest_version() ? false : true,
+			'should_track_logged_in_users' => false,
```

Comment:
This has changed since 3.5; `track_authenticated_users` was `true` in 3.5 and changed to `false` later.